src/olympia/amo/management/commands/compress_assets.py

```diff
@@ -0,0 +1,302 @@
+
+        if options.get('do_update_only', False):
+            self.update_hashes(update=True)
+            return
+
+        jar_path = (os.path.dirname(__file__), '..', '..', 'bin',
+                    'yuicompressor-2.4.7.jar')
+        self.path_to_jar = os.path.realpath(os.path.join(*jar_path))
+
+        self.v = '-v' if options.get('verbosity', False) == '2' else ''
+
+        cachebust_imgs = getattr(settings, 'CACHEBUST_IMGS', False)
+        if not cachebust_imgs:
+            print 'To turn on cache busting, use settings.CACHEBUST_IMGS'
+
+        # This will loop through every bundle, and do the following:
+        # - Concat all files into one
+        # - Cache bust all images in CSS files
+        # - Minify the concatted files
+
+        for ftype, bundle in settings.MINIFY_BUNDLES.iteritems():
+            for name, files in bundle.iteritems():
+                # Set the paths to the files.
+                concatted_file = path(ftype, '%s-all.%s' % (name, ftype,))
+                compressed_file = path(ftype, '%s-min.%s' % (name, ftype,))
+
+                files_all = []
+                for fn in files:
+                    processed = self._preprocess_file(fn)
+                    # If the file can't be processed, we skip it.
+                    if processed is not None:
+                        files_all.append(processed)
+
+                # Concat all the files.
+                tmp_concatted = '%s.tmp' % concatted_file
+                if len(files_all) == 0:
+                    raise CommandError(
+                        'No input files specified in '
+                        'MINIFY_BUNDLES["%s"]["%s"] in settings.py!' %
+                        (ftype, name)
+                    )
+                self._call(
+                    'cat %s > %s' % (' '.join(files_all), tmp_concatted),
+                    shell=True
+                )
+
+                # Cache bust individual images in the CSS.
+                if cachebust_imgs and ftype == "css":
+                    bundle_hash = self._cachebust(tmp_concatted, name)
+                    self.bundle_hashes["%s:%s" % (ftype, name)] = bundle_hash
+
+                # Compresses the concatenations.
+                is_changed = self._is_changed(concatted_file)
+                self._clean_tmp(concatted_file)
+                if is_changed or not os.path.isfile(compressed_file):
+                    self._minify(ftype, concatted_file, compressed_file)
+                elif self.v:
+                    print 'File unchanged, skipping minification of %s' % (
+                        concatted_file)
+                else:
+                    self.minify_skipped += 1
+
+        # Write out the hashes
+        self.update_hashes(options.get('add_timestamp', False))
+
+        if not self.v and self.minify_skipped:
+            print 'Unchanged files skipped for minification: %s' % (
+                self.minify_skipped)
```

Comment:
can we wrap all `print xxx` as `print(xxx)` so it's automagically python3 compatible.